pioinstaller/util.py

```diff
@@ -96,3 +106,19 @@ def unpack_archive(src, dst, mode="r:gz"):
 
 def get_installer_script():
     return os.path.abspath(sys.argv[0])
+
+
+def get_systype():
+    type_ = platform.system().lower()
+    arch = platform.machine().lower()
+    if type_ == "windows":
+        arch = "amd64" if platform.architecture()[0] == "64bit" else "x86"
+    return "%s_%s" % (type_, arch) if arch else type_
+
+
+def safe_clean_dir(path, raise_exception=False):
```

Comment:
rename to `safe_remove_dir` because it is not clear that this fucntion remove fodler.